src/ex4/dist/Ui.js

```diff
@@ -25,16 +25,16 @@ class Ui {
         this.duplicateListItemRemover(ul);
         const items = await this.RemoteItemManager.get();
         items.forEach(element => {
-            const li = document.createElement('li'); 
-            if (element.done) {
+            const li = document.createElement('li');
+            if (element.status) {
```

Comment:
why status=true means done?
if its called "done" it make sense that its a boolean flag like before
but if its status it should probably use an enum indicating the status, and then here you can check if status === done